### PR TITLE
fix: improve editor and plugin part

### PR DESCRIPTION
**Pull request by nodece**

### CODE REVIEW

File: website/pages/en/editor-integration.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const React = require('react');
```
Comment (hsluoyz) [inline website/pages/en/editor-integration.js:1]: Why not directly add a docs page for plugins? I think it's better than a custom React page.

Comment (nodece) [reply website/pages/en/editor-integration.js:1]: @hsluoyz Good point. How about change the editor to playground on navbar?

docs page structure:
-  Editors
    - Online Editor
    - Editor Integration

Comment (hsluoyz) [reply website/pages/en/editor-integration.js:1]: @nodece I'm OK to rename Online Editor to Playground, but I think we should unify its naming, including the official site homepage, docs page, and even the GitHub repo: https://github.com/casbin/casbin-editor

image

For `Editor Integration`, I think maybe `IDE Plugins` would be more straightforward?

Comment (nodece) [reply website/pages/en/editor-integration.js:1]: ok

**Comment by hsluoyz** [comment]
@nodece have you checked if this works?

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**